frontend/packages/helm-plugin/src/topology/sidebar/release-panel/tab-sections.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { GraphElement, Node } from '@patternfly/react-topology';
+import { useTranslation } from 'react-i18next';
+import { StatusBox } from '@console/internal/components/utils/status-box';
+import TopologyGroupResourcesPanel from '@console/topology/src/components/side-bar/TopologyGroupResourcesPanel';
+import { getResource } from '@console/topology/src/utils';
+import HelmReleaseOverview from '../../../components/details-page/overview/HelmReleaseOverview';
+import { TYPE_HELM_RELEASE } from '../../components/const';
+import TopologyHelmReleaseNotesPanel from '../../TopologyHelmReleaseNotesPanel';
+
+export const useHelmReleasePanelDetailsTabSection = (element: GraphElement) => {
```

Comment:
We're going to have a lot of sections.
The first check must be made efficiently to filter out all the sections which do not apply to the given element.
These should not be rendered as a hook.

After successfully passing the check, they should return a component which itself can use hooks within.